### ISSUE TITLE
XOR Interpretation

### ISSUE THREAD

**Issue by seanlaw** [codekansas/pydata-carolinas-2016#1]
Shouldn't [0 1] and [1 0] be True?  

https://en.wikipedia.org/wiki/Exclusive_or#Truth_table

In both Theano and Keras XOR notebooks it looks like you have them flipped.

**Comment by codekansas** [comment]
It's the same thing basically, I guess really xnor. They're both not
linearly separable

**Comment by seanlaw** [comment]
I agree that the problem is essentially the same. However, the name of the notebooks (XOR) contradicts the code (XNOR). My OCD is getting the best of me so feel free to ignore :)
